### PR TITLE
Adapt angular ngeo.message

### PR DESCRIPTION
**Pull request by pfirpfel**

### CODE REVIEW

File: examples/modal.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
 /** @suppress {extraRequire} */
-goog.require('ngeo.modalDirective');
+goog.require('ngeo.message.modalComponent');
 
 
 /** @type {!angular.Module} **/
-app.module = angular.module('app', ['ngeo']);
+app.module = angular.module('app', [
+  'ngeo',
```
Comment (ger-benjamin) [inline examples/modal.js]: `ngeo.module.name` is better no ?
Look also other ngeo example (`git grep \'ngeo\'`)

File: src/message/Message.js [hunk 2]
```diff
@@ -153,3 +125,31 @@ ngeo.Message.prototype.getMessageObjects = function(object, opt_type) {
 
   return msgObjects;
 };
+
+
+/**
+ * @enum {string}
+ * @export
+ */
+ngeo.message.Message.MessageType = {
```
Comment (ger-benjamin) [inline src/message/Message.js]: That's a little bit repetitive for a name...
Can you still change it to `ngeo.message.Message.type` ?
(And the last name should start without uppercase no ?)

Comment (pfirpfel) [reply src/message/Message.js]: Should start with a uppercase character. Google does it [this way](https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming).

How about `ngeo.message.Message.Type`?

Comment (ger-benjamin) [reply src/message/Message.js]: okay

Comment (pfirpfel) [reply src/message/Message.js]: Changed it

File: examples/modal.js (hunk 1 above)
Comment (pfirpfel) [reply examples/modal.js]: This should be all of them.

**Comment by ger-benjamin** [comment]
I've restarted the build, it was on error after a timeout.

**Pull request merged**